moviepy/video/VideoClip.py

```diff
@@ -1040,7 +1040,7 @@ def __init__(
 
         if not isinstance(img, np.ndarray):
             # img is a string or path-like object, so read it in from disk
-            img = imread(img)
+            img = imread(img, index=0)[0]
```

Comment:
```suggestion
            img = imread(img, index=0)
```

If you add a slice to the call you will get the first row of the image. I suppose you meant to just get the first frame/image in the file?